README.md

```diff
@@ -1,2 +1,41 @@
 # Awesome Book
```

Comment:
- [OPTIONAL]. You haven't yet added this summary of this project as requested by the previous code reviewer. Will you do it now?. This will boost your README file.